src/core_plugins/console/index.js

```diff
@@ -150,14 +151,13 @@ module.exports = function (kibana) {
         path: '/api/console/api_server',
         method: ['GET', 'POST'],
         handler: function (req, reply) {
-          let server = require('./api_server/server');
-          let { sense_version, apis } = req.query;
+          const { senseVersion, apis } = req.query;
```

Comment:
I'm going to change it back since it's pulled from the request.

To be honest, I'm surprised this change worked at all, so I suspect there's some unused code/behaviors here.  In any case, that's beyond the scope of this change, so I'd prefer to keep this to purely internal changes.